docs/pages/validator-management/vc-configuration.md

```diff
@@ -88,11 +88,31 @@ Configure your validator client's fee recipient address by using the `--suggeste
 
 You may choose to use the `--strictFeeRecipientCheck` flag to enable a strict check of the fee recipient address with the one returned by the beacon node for added reassurance.
 
+### Configure your builder selection and/or builder boost factor
+
+If you are running a beacon node with connected builder relays, you may use these validator configurations to signal which block (builder vs. local execution) the beacon node should produce.
+
+With produceBlockV3 (enabled automatically after the Deneb hard fork), the `--builder.boostFactor` is a percentage multiplier the block producing beacon node must apply to boost (>100) or dampen (<100) builder block value for selection against execution block. The multiplier is ignored if `--builder.selection` is set to anything other than `maxprofit`. Even though this is set on the validator client, the calculation is applied on the beacon node itself. For more information, see the [produceBlockV3 Beacon API](https://ethereum.github.io/beacon-APIs/#/ValidatorRequiredApi/produceBlockV3).
+
+With Lodestar's `--builder.selection` options, you can select:
+
+- `maxprofit`: Default setting for Lodestar set at `--builder.boostFactor=100`. This default setting will always choose the more profitable block. Using this option, you may customize your `--builder.boostFactor` to your preference. Examples of its usage are below.
+- `executiononly`: Validator will not select the builder block even if builder relays are configured on the beacon node. This option will always select the local execution block.
+- `builderalways`: An alias of `--builder.boostFactor=18446744073709551615` (2**64 - 1), which will select the builder block, unless the builder block fails to produce. The builder block may fail to produce if it's not available, not timely or there is an indication of censorship via shouldOverrideBuilder from the execution payload response.
+- `builderonly`: Generally used for distributed validators (DVs). No execution block production will be triggered. Therefore, if a builder block is not produced, the API will fail and _no block will be produced_.
+
+#### Examples using builder boost factor
+
+Example 1: Setting a `--builder.boostFactor=0` will always prefer the local execution block, but will produce an available builder block if the local execution block fails.
+
+Example 2: Setting a `--builder.boostFactor=100` is the same as signaling `--builder.selection maxprofit` where the validator will always select the most profitable block between the local execution engine and the builder block from the relay.
+
+Example 3: Setting `--builder.boostFactor=80` allows you to signal that your builder block bid needs 20% more in value comparatively to your local execution block to be selected.
```

Comment:
Just noticed, the example is not correct.

Let's assume a local block value of 100, if builder block has value of 120 (20% more) it will still pick local block

> 120 * 0.80 = 96

Reply:
its inverse relationship so need 25% more (100/0.8 -100)